File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        d = {}
        for i in nums:
            d[i] = d.get(i,0)+1
        print(d)
        m = 0
        for i in d:
            if not d.get(i-1,0):
                j = i
                while d.get(j,0):
                    j+=1
                if j-i>m:
                    m = j-i
        return m 
                

File: Data Structures & Algorithms/duplicate-integer/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        const d = {}
        for(let i=0; i<nums.length; i++){
            if (d[nums[i]] == 1) return true
            else d[nums[i]] = 1
        }
        return false
    }
}
